src/util.ts

```diff
@@ -727,13 +753,41 @@ export class Util {
     config: MakeRequestConfig,
     callback: BodyResponseCallback
   ): void | Abortable {
+    let autoRetryValue = AUTO_RETRY_DEFAULT;
+    if (
+      config.autoRetry !== undefined &&
+      config.retryOptions?.autoRetry !== undefined
+    ) {
+      throw new ApiError(
```

Comment:
If I'm understanding this logic, we through if a customer sets both `autoRetry` and `retryOptions.autoRetry`?

Is there any risk that when this is deployed existing customer applications are somehow setting both these parameters?

Reply:
All applications using nodejs-common should not be using these fields because they did not exist prior to this PR. Storage handles this case directly https://github.com/googleapis/nodejs-storage/pull/1474/files